packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -163,6 +177,7 @@ const TextareaField = (props) => {
     >
       <Textarea
         {...inputAttrs}
+        className={classes}
```

Comment:
Thing I said before about inputAttrs

Reply:
No longer an issue now since `isIncomplete` works more similar to the `isInvalid` prop and the Input control itself adds the class in. https://github.com/cerner/terra-core/pull/2670/commits/93158da0ccc2be1d72588890924879e913215a96